demo/demo-spring-boot-transport/demo-spring-boot-pojo-client/src/main/java/io/servicecomb/demo/pojo/client/PojoClientTest.java

```diff
@@ -105,7 +110,7 @@ private static void testEmpty(Test test) {
 
   private static void testNull(Test test) {
     TestMgr.check("code is 'null'", test.getTestString(null));
-    TestMgr.check(null, test.wrapParam(null));
+//    TestMgr.check(null, test.wrapParam(null));
```

Comment:
lost this test case.

Reply:
I commented out this line for debugging... I will retrieve it.